moveit_core/planning_scene/test/test_planning_scene.cpp

```diff
@@ -57,8 +57,22 @@ void loadRobotModel(urdf::ModelInterfaceSharedPtr& robot_model_out)
     xml_string += (line + "\n");
   }
   xml_file.close();
+  file_content = xml_string;
+}
+
+void loadRobotModel(urdf::ModelInterfaceSharedPtr& robot_model_out)
+{
+  std::string xml_string;
+  loadModelFile("pr2_description/urdf/robot.xml", xml_string);
   robot_model_out = urdf::parseURDF(xml_string);
 }
+void loadRobotModels(urdf::ModelInterfaceSharedPtr& robot_model_out, srdf::ModelSharedPtr& srdf_model_out)
+{
+  loadRobotModel(robot_model_out);
+  std::string xml_string;
+  loadModelFile("pr2_description/srdf/robot.xml", xml_string);
+  srdf_model_out->initString(*robot_model_out, xml_string);
```

Comment:
Ideally all tests would use the same test utilities for loading URDFs / SRDFs instead of remunerating them in every test. I have the same suggestion for a test @mlautman is working on in fact.

MoveIt! already has common file loading functionality for URDF / SRDF [here](https://github.com/ros-planning/moveit/blob/kinetic-devel/moveit_ros/planning/robot_model_loader/include/moveit/robot_model_loader/robot_model_loader.h#L61) - can that be reused in this file?